domain/schema/model/sql/0022-resource.sql

```diff
@@ -7,7 +7,7 @@ CREATE UNIQUE INDEX idx_resource_origin_name
 ON resource_origin_type (name);
 
 INSERT INTO resource_origin_type VALUES
-(0, 'uploaded'),
+(0, 'upload'),
```

Comment:
I disagree with this change. A resource has been uploaded or taken from a store. Upload as word doesn't fit to me.

Reply:
Note: Code removed (necessary for add, not used in delete). Anyway having a consensus on the topic would be useful.